WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md

```diff
@@ -9,43 +9,44 @@ author: MicrosoftGuyJFlo
 
 Protecting privileged access against determined adversaries requires you to take a complete and thoughtful approach to isolate these systems from risks.
 
 ## What are privileged accounts?
 
-Before we talk about how to secure them lets define privileged accounts.
+Before we talk about how to secure them, lets define privileged accounts.
 
-Privileged Accounts like administrators of Active Directory Domain Services have direct or indirect access to most or all assets in an IT organization, making a compromise of these accounts a significant business risk.
+Privileged Accounts, like administrators of Active Directory Domain Services, have direct or indirect access to most or all assets in an IT organization, making a compromise of these accounts a significant business risk.
 
-## Why securing privileged access is important?
+## Why is securing privileged access important?
 
 Cyber-attackers focus on privileged access to systems like Active Directory (AD) to rapidly gain access to all of an organizations targeted data. Traditional security approaches have focused on the network and firewalls as the primary security perimeter, but the effectiveness of network security has been significantly diminished by two trends:
 
-* Organizations are hosting data and resources outside the traditional network boundary on mobile enterprise PCs, devices like mobile phones and tablets, cloud services, and bring your own devices (BYOD)
+* Organizations are hosting data and resources outside the traditional network boundary on mobile enterprise PCs, devices like mobile phones and tablets, cloud services, and bring your own devices (BYOD).
 * Adversaries have demonstrated a consistent and ongoing ability to obtain access on workstations inside the network boundary through phishing and other web and email attacks.
 
-These factors necessitate building a modern security perimeter out of authentication and authorization identity controls in addition to the traditional network perimeter strategy. A security perimeter here is defined as a consistent set of controls between assets and the threats to them. Privileged accounts are effectively in control of this new security perimeter so it's critical to protect privileged access.
+These factors necessitate building a modern security perimeter out of authentication and authorization identity controls, in addition to the traditional network perimeter strategy. A security perimeter here is defined as a consistent set of controls between assets and the threats to them. Privileged accounts are effectively in control of this new security perimeter, so it is critical to protect privileged access.
```

Comment:
@illfated  We actually use contractions in our content now as part of our updated voice and tone. If you could revert that single change back to it's, I'd appreciate it.

Reply:
Fair enough. I presume my suggestion of removing the contraction comes from my experience with British English.

Reverted, as requested.